website/pages/docs/plugins/sources/azure/_configuration.md

```diff
@@ -16,4 +16,9 @@ spec:
     # skip_subscriptions: []
     # normalize_ids: false
     # oidc_token: ""
+    # retry_options:
+    #   max_retries: 3
+    #   try_timeout_seconds: 0
+    #   retry_delay_seconds: 4
+    #   max_retry_delay_seconds: 60
```

Comment:
Did not add `status_codes` as I think that's quite an advanced option and also I'd have to hard code the default codes from the Azure SDK